### PR TITLE
Feature/manage layouts api

### PR DESCRIPTION
**Pull request by Eazybright**

### CODE REVIEW

File: src/Actions/ManagesLayout.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    public function deleteLayout(string $layoutId)
+    {
+        return $this->delete("layouts/$layoutId");
+    }
+
+    /**
+     * Update a layout
+     * Update the name, content, description, identifier and variables of a layout. 
+     * 
+     * @param array $layoutId
```
Comment (unicodeveloper) [inline src/Actions/ManagesLayout.php:64]: This should be a string

File: README.md [hunk 2]
```diff
@@ -325,6 +325,38 @@ $novu->deleteIntegration($integrationId);
...
+// Update layout
+$novu->updateLayout('<insert->layout->id', [
+    'name' => '<insert->name->of->layout>',
+    'identifier' => '<insert->identifier>',
+    'content' => '<insert->html->content>',
+])->toArray();
+
+// Delete layout
+$novu->deleteLayout('<insert->layout->id');
+
```
Comment (unicodeveloper) [inline README.md:357]: Please change the `->` in `<insert->name->of->layout>` and the other occurrences to `-`

**Review by unicodeveloper — COMMENTED** [review]
Thank you so much for your contribution. Please attend to the comments. 🙏🏿

**Comment by Eazybright** [comment]
> Thank you so much for your contribution. Please attend to the comments. 🙏🏿

PR updated

File: src/Resources/Layout.php [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+    /**
+     * The variables.
+     *
+     * @var array
+     */
+    public $variables;
+
+    /**
+     * Return the array form of NotificationTemplate object.
```
Comment (unicodeveloper) [inline src/Resources/Layout.php:120]: NotificationTemplate should be Layout

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**